src/main/java/com/puppycrawl/tools/checkstyle/utils/CommonUtils.java

```diff
@@ -101,6 +113,112 @@ public static Pattern createPattern(String pattern, int flags) {
     }
 
     /**
+     * Creates DetailAST block comment to pass it to the Javadoc parser.
+     * @param content comment content.
+     * @return DetailAST block comment
+     */
+    public static DetailAST createFakeBlockComment(String content) {
```

Comment:
Both moved methods seem to have the same purpose, but different parameters - can we use overloading here?

Reply:
fixed